src/pybind/mgr/dashboard/frontend/src/app/shared/components/custom-login-banner/custom-login-banner.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+import _ from 'lodash';
+
+import { CustomLoginBannerService } from '~/app/shared/api/custom-login-banner.service';
+
+@Component({
+  selector: 'cd-custom-login-banner',
+  templateUrl: './custom-login-banner.component.html',
+  styleUrls: ['./custom-login-banner.component.scss']
+})
+export class CustomLoginBannerComponent implements OnInit {
+  customLoginBannerText: string;
+  constructor(private customLoginBannerService: CustomLoginBannerService) {}
+
+  ngOnInit(): void {
+    this.customLoginBannerService.getBannerText().subscribe((response: string) => {
+      if (response) {
+        this.customLoginBannerText = response;
+      }
+    });
```

Comment:
Since the introduction of the AsyncPipe in Angular, it favors the following syntax for consuming observables from the HTML template (the trailing `$` is a convention for observables):
```suggestion
    this.bannerText$ = this.customLoginBannerService.getBannerText();
```
And in the HTML side:
```html
<p class="p-3 m-0 font-weight-bold"
   *ngIf="bannerText$ | async as bannerText">{{ bannerText }}</p>
```
Super-clean: you don't have to unsubscribe (the AsyncPipe takes care for you), nor work with closures. It almost looks like classic synchronous/procedural code, rather than asynchronous/reactive one.

Reply:
Hmm, looks neat!!